pkg/controller/backup_sync_controller.go

```diff
@@ -171,6 +172,27 @@ func (b *backupSyncReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		}
 		backup.Labels[velerov1api.StorageLocationLabel] = label.GetValidName(backup.Spec.StorageLocation)
 
+		//check for the backup schedule. If it does not exist, remove it.
+		listedReferences := backup.ObjectMeta.OwnerReferences
+		foundReferences := make([]metav1.OwnerReference, 0)
```

Comment:
How about only checking the schedule owner reference explicitly?
Although the schedule is the only owner of the backup currently, we may add more in the future.

Reply:
do you mean just checking the owner reference type and only checking the schedules? if so I can add that yeah.

Reply:
Yes